### DISCUSSION TITLE
Anticipated Behavior for an Incorrect Docker0 IP Address

### DISCUSSION THREAD

**Discussion by classzhou** [Security-Onion-Solutions/securityonion#11266] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 core

### RAM

32 GB

### Storage for /

1024 T

### Storage for /nsm

720 G

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm an amateur enthusiast in the field of network security, and I've recently been trying to use Security Onion to pursue my hobby. Unfortunately, my network has been in a rather delicate and unreachable state. Here's my network configuration for Security Onion 2.4.10:

## MGMT:

- IP Address: 172.17.65.200/23
- Gateway: 172.17.65.254

My laptop connects to Security Onion via VPN
## VPN Internal Network Mapping Address:

- IP Address: 172.17.0.230/24

## Security Onion Docker Subnet:

- IP Address: 192.168.100.0/24

However, there seems to be an issue with the Docker0 IP address, which is set to 172.17.0.1. This has resulted in my laptop being unable to connect to Security Onion. Is this behavior of the Docker0 address expected?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Have you tried changing the Docker IP range during Setup?
https://docs.securityonion.net/en/2.4/docker.html?#networking-and-bridging
